pynest/examples/iaf_tum_2000_short_term_depression.py

```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
+#
+# iaf_tum_2000_short_term_depression.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+r"""
+Short-term depression example
+-----------------------------
+
+The `iaf_tum_2000 <../models/iaf_tum_2000>` neuron [1]_ is a model with
+*short-term synaptic plasticity*. Short-term plasticity can either strengthen
+or weaken a synapse and acts on a timescale of milliseconds to seconds. This
+example illustrates *short-term depression*, which is caused by depletion of
+the pool of readily releasable neurotransmitters at the axon terminal of a
+presynaptic neuron when many presynaptic spikes occur in rapid succession.
+During synaptic depression, the strength of the synapse declines until this
+pool can be replenished.
+
+In the ``iaf_tum_2000`` model, a fraction :math:`u` of the available synaptic
+resources :math:`x` is used by each presynaptic spike (see Eq. 3 and 4 or
+Eq. 2.1 and 2.2 in [1]_ or [2]_, respectively). A parameter :math:`U \in [0, 1]`
+determines the increase in :math:`u` with each spike.
+
+In this example, we reproduce Figure 1A in [2]_. We connect two
+``iaf_tum_2000`` neurons. The presynaptic neuron is driven by DC input and
+we record the voltage trace of the postsynaptic neuron. Short-term depression
+is enabled by setting :math:`U` to a large value, which causes a fast saturation
+of the synaptic efficacy.
+
+For an example on *short-term facilitation*, see
+:doc:`../auto_examples/iaf_tum_2000_short_term_facilitation`.
+
+.. note::
+
+    The `iaf_tum_2000 <../models/iaf_tum_2000>` neuron model incorporates the
+    `tsodyks_synapse <../models/tsodyks_synapse>` computations directly in
+    `iaf_psc_exp <../models/iaf_psc_exp>` on the presynaptic side, which
+    makes ``iaf_tum_2000`` more computationally efficient.
```

Comment:
I think the following will be more informative for users:

```suggestion
    The `iaf_tum_2000 <../models/iaf_tum_2000>` neuron model combined with 
    `static_synapse` provides a more efficient implementation of the model studied
    in [1]_ and [2]_ than the combination of `iaf_psc_exp <../models/iaf_psc_exp>` 
    with `tsodyks_synapse <../models/tsodyks_synapse>`.
```